packages/react/src/components/card-messages/CardMessages.tsx

```diff
@@ -52,32 +58,26 @@ export const CardMessages = ({ items, user, updateMessagesCount }: { items: Mess
     setMessage('');
     updateMessagesCount(messages.length + 1);
   }, [message, title, messages, user, updateMessagesCount]);
-  const cancel = useCallback(() => {
-    setTitle('');
-    setMessage('');
-  }, []);
+
   const onTitleChanged = useCallback((value) => {
```

Comment:
Remove unnecessary useCallback
https://stackoverflow.com/questions/66844519/send-setstate-from-usestate-as-a-props-using-usecallback#:~:text=You%20don't%20need%20to%20wrap%20setState%20in%20use%20callback,doesn't%20change%20hook%20setState.